grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/MinMapReduce.java

```diff
@@ -34,14 +34,14 @@
  * @author Sheldon Hall
  */
 
-public class MinMapReduce extends GraknMapReduce<Number> {
+public class MinMapReduce extends StatisticsMapReduce<Number> {
 
     // Needed internally for OLAP tasks
     public MinMapReduce() {
     }
 
-    public MinMapReduce(Set<TypeId> selectedTypeIds, String resourceDataType) {
-        super(selectedTypeIds, resourceDataType);
+    public MinMapReduce(Set<TypeId> selectedTypeIds, String resourceDataType, String degreeKey) {
```

Comment:
if `resourceDataType` does refer to the datatype of a resource type then this maybe should be `ResourceType.DataType dataType`. Again it just makes things more explicit.